src/energyplus/ForwardTranslator/ForwardTranslateGroundHeatExchangerVertical.cpp

```diff
@@ -166,28 +176,12 @@ namespace energyplus {
       propertiesIdfObject.setDouble(GroundHeatExchanger_Vertical_PropertiesFields::UTubeDistance, value.get());
     }
 
-    IdfObject groundIdfObject(IddObjectType::Site_GroundTemperature_Undisturbed_KusudaAchenbach);
-    m_idfObjects.push_back(groundIdfObject);
-
-    groundIdfObject.setName(groundModelName);
-
-    if ((value = modelObject.groundThermalConductivity())) {
-      groundIdfObject.setDouble(Site_GroundTemperature_Undisturbed_KusudaAchenbachFields::SoilThermalConductivity, value.get());
-    }
-
-    groundIdfObject.setDouble(Site_GroundTemperature_Undisturbed_KusudaAchenbachFields::SoilDensity, 920.0);
+    // UndisturbedGroundTemperatureModelName
+    idfObject.setString(GroundHeatExchanger_SystemFields::UndisturbedGroundTemperatureModelName, s.get());
 
-    if ((value = modelObject.groundThermalHeatCapacity())) {
```

Comment:
What happens to these fields if they are set on the GroundHeatExchangerVertical system? If looks like they might be ignored. Maybe the implementation of these methods like GroundHeatExchangerVertical::groundThermalHeatCapacity() should map to the contained ground temperature model?

Reply:
`ghxVert.groundThermalConductivity()` and `ghxVert.groundThermalHeatCapacity()` both still get translated above.

`ghxVert.groundTemperature()` previously was only translated to the `groundTempModel`. `AverageSoilSurfaceTemperature` for the kusuda object is now handled by its own `averageSoilSurfaceTemperature()` method.

This is basically the same approach we used for the `HorizontalTrench` implementation - the `ghxVert` gets the `soilThermalConductivity()` and `soilSpecificHeat`, and then kusuda FTs its own `averageSoilSurfaceTemperature()`.

Reply:
sorry missed the above. Got it.